### ISSUE TITLE
perforce integration seems to be broken

### ISSUE THREAD

**Issue by spar01** [mhinz/vim-signify#266]
Just updated the plugin and perforce integration of the plugin seems to be broken as of commit cc2b17d462c9460c2f5f4a0d71a3edf6e6a71987

SignifyList shows the following:
```
    active  =  1                                                                                                                                                                              
    buffer  =  3                                                                                                                                                                              
       vcs  =  []                                                                                                                                                                             
     stats  =  -1 added, -1 changed, -1 removed                                                                                                                                               
    signid  =  256                                                                                                                                                                            
     hunks  =  []    
```

also SignifyDebug comes up empty.

Reverting to commit a9fc705b9bdffaac46f13e47d6565c904102dedc I see

```
    active  =  1                                                                                                                                                                              
    buffer  =  3                                                                                                                                                                              
       vcs  =  ['perforce']                                                                                                                                                                   
     stats  =  6 added, 0 changed, 0 removed                                                                                                                                                  
    signid  =  262                                                                                                                                                                            
     hunks  =  start: 585, end: 590, IDs: [256, 257, 258, 259, 260, 261]   
```

**Comment by spar01** [comment]
I can see the expected diffs

**Comment by mhinz** [comment]
Do `vim -V1 file` while `file` is any file with changes. Then paste all lines starting with `[sy` from `:messages`.

**Comment by spar01** [comment]
I think I might have found the issue. I see the commit added a check for win32, but did not carry over the '&&'.

Looks like the command is getting interpreted as:
`p4 info >/dev/null 2>&1 env P4DIFF= P4COLORS= p4 diff -du0`

instead of:
`p4 info >/dev/null 2>&1 && env P4DIFF= P4COLORS= p4 diff -du0`

Fixing the above, fixed the issue for me.

**Comment by spar01** [comment]
I have gnerated a pull request for the fix #267
